modules/git/object_format.go

```diff
@@ -13,6 +13,14 @@ var sha1Pattern = regexp.MustCompile(`^[0-9a-f]{4,40}$`)
 
 const Sha1ObjectFormatName = "sha1"
 
+func IsValidObjectFormat(name string) bool {
+	switch name {
```

Comment:
This could be just `return ObjectFormatFromName(name) != nil` so that we don't have this logic in multiple places

Reply:
updated on 3db18697bbab039097287c6f73045073a1983c8f